charts/prefect-server/templates/background-services/deployment.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
   {{- end }}
 spec:
   revisionHistoryLimit: {{ .Values.backgroundServices.revisionHistoryLimit }}
-  replicas: 1
+  replicas: {{ .Values.backgroundServices.replicaCount }}
```

Comment:
(no action) just highlighting that this is no longer hard-coded to `1`

Reply:
Unless we support more than 1 simultaneous background service worker, then I should revert this.

Reply:
We can check with @zzstoatzz (ref: https://github.com/PrefectHQ/prefect-helm/commit/dd79375e9c7b9d26aae3f8358bdc629102bd3f93)

Reply:
https://github.com/PrefectHQ/prefect/issues/18753